src/components/measureLanding/measureList/MeasureList.tsx

```diff
@@ -816,6 +816,34 @@ export default function MeasureList(props: {
     setOpenAssociateCmsIdDialog(true);
   };
 
+  const handleCmsIdAssociationContinueDialog = (
```

Comment:
Not a big deal but the naming could be simplified

Reply:
I'll change it to handleCmsIdAssociation